### ISSUE TITLE
[Feature] Retrieving run stats from HTML report via command line

### ISSUE THREAD

**Issue by nbaldzhiev** [microsoft/playwright#28423]
I currently have the case where our tests run in parallel across multiple run jobs, which then get combined into a unified HTML Playwright report. I am trying to gather the stats of the run, i.e. number of all, passed, failed, flaky, skipped tests so that I can have that information and use it to send a Slack notification, for instance. However, I am not able to parse the `index.html` in any way that would allow me to get that data.

I realise that such feature request might be of low priority, but essentially what I am looking for is a way, perhaps an API, through which I can get the run stats from the HTML report. This can save a lot of effort, which would otherwise go to gathering the stats from each run job output and then making a sum of each all/passed/failed/flaky/skipped numbers.

I tried numerous ways of getting that info from the HTML report, but I couldn't. I guess I couldn't understand how the report calculates that. And of course, scraping is an option, but I was wondering if we can go without it.

**Comment by StasGrishaevTutu** [comment]
It seems that instead of parsing the html file it is better to use another type of reporter and send the data received through it, doesn’t it? for example via [json reporter](https://playwright.dev/docs/test-reporters#json-reporter)

**Comment by nbaldzhiev** [comment]
> It seems that instead of parsing the html file it is better to use another type of reporter and send the data received through it, doesn’t it? for example via [json reporter](https://playwright.dev/docs/test-reporters#json-reporter)

We use the HTML report a lot, so I am trying to reuse it for this purpose here as well, so that I can collect the stats and present them together with a link to the HTML report itself.

**Comment by pavelfeldman** [comment]
> We use the HTML report a lot, so I am trying to reuse it for this purpose here as well

You can have as many reporters in parallel at a time. If you are using blob report to merge into single HTML, the same will work for non-HTML reporter.
